examples/all-clusters-app/all-clusters-common/src/binding-handler.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "binding-handler.h"
+
+#include "app-common/zap-generated/command-id.h"
```

Comment:
This header is not used...  But `app-common/zap-generated/ids/Commands.h` _is_ used but not directly included...